packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl

```diff
@@ -1,5 +1,6 @@
-verificationReminderSecond-subject = Final reminder: Activate your account
-verificationReminderSecond-title = Still there?
-verificationReminderSecond-description-2 = Almost a week ago you created a { -product-firefox-account } but never confirmed it. We’re worried about you.
+verificationReminderSecond-subject-2 = Remember to confirm your account
+verificationReminderSecond-title-2 = Don't miss out on { -brand-firefox }
```

Comment:
```suggestion
verificationReminderSecond-title-2 = Don’t miss out on { -brand-firefox }
```

Reply:
Thanks @pdehaan for catching the typographic quotes.